File: Examples/CSharp/Articles/WorkingWithHTMLFormat/ExpandTextFromRightToLeft.cs
using System;
using System.IO;
using Aspose.Cells;

namespace Aspose.Cells.Examples.CSharp.Articles.WorkingWithHTMLFormat
{
    public class ExpandTextFromRightToLeft
    {
        public static void Run()
        {
            //Source directory
            string sourceDir = RunExamples.Get_SourceDirectory();

            //Output directory
            string outputDir = RunExamples.Get_OutputDirectory();
           
            // Load source excel file inside the workbook object
            Workbook wb = new Workbook(sourceDir + "sampleExpandTextFromRightToLeft.xlsx");

            // Save workbook in html format
            wb.Save(outputDir + "outputExpandTextFromRightToLeft_" + CellsHelper.GetVersion() + ".html", SaveFormat.Html);

            Console.WriteLine("ExpandTextFromRightToLeft executed successfully.");
        }
    }
}

File: Examples/VisualBasic/Articles/WorkingWithHTMLFormat/ExpandTextFromRightToLeft.vb
Imports System.IO
Imports Aspose.Cells

Namespace Articles.WorkingWithHTMLFormat
    Public Class ExpandTextFromRightToLeft
        Public Shared Sub Run()
            ' ExStart:1
            ' The path to the documents directory.
            Dim dataDir As String = RunExamples.GetDataDir(System.Reflection.MethodBase.GetCurrentMethod().DeclaringType)

            ' Load source excel file inside the workbook object
            Dim wb As New Workbook(dataDir & "sample.xlsx")

            ' Save workbook in html format
            wb.Save(dataDir & "ExpandTextFromRightToLeft_out_" + CellsHelper.GetVersion() + ".html", SaveFormat.Html)
            ' ExEnd:1           

        End Sub
    End Class
End Namespace

